blockchains/avalanche/assets/0xC891EB4cbdEFf6e073e859e987815Ed1505c2ACD/info.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Euro Coin",
+  "website": "https://www.circle.com/en/euro-coin",
+  "description": "Euro Coin, or EUROC, is a euro-backed stablecoin that’s accessible globally on Ethereum and Avalanche.",
+  "socials": [],
+  "explorer": "https://snowtrace.io/address/0xC891EB4cbdEFf6e073e859e987815Ed1505c2ACD",
```

Comment:
@schystz the one above is /token, this one is /address. I guess it doesn't matter?